.github/workflows/clean-rev-image.yml

```
@@ -0,0 +1,96 @@
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the code
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    # Set up Docker for Docker Hub login
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)